prow/cluster/gcsweb/README.md

```diff
@@ -0,0 +1,44 @@
+# Info
```

Comment:
Let's remove this document and proceed this way:

Let's add a small section here: https://github.com/kyma-project/test-infra/blob/master/docs/prow/prow-architecture.md#components and call it `gcsweb` (similar to, for example, Branch Protector). 
The section will contain a simple overview and a link to further resources. For example:

### gcsweb

gcsweb is a lightweight web frontend for GCS which allows you to access the content of the **artifacts** tab in Spyglass without the need to log in. For more information on gcsweb read [this]( https://github.com/kubernetes/k8s.io/tree/master/gcsweb.k8s.io) document.

(as I posted in the comment)

Reply:
done